pkg/server/server.go

```diff
@@ -359,9 +359,13 @@ func (s *Server) Check(ctx context.Context, req *openfgav1.CheckRequest) (*openf
 		TupleKey:             req.GetTupleKey(),
 		ContextualTuples:     req.ContextualTuples.GetTupleKeys(),
 		ResolutionMetadata: &graph.ResolutionMetadata{
-			Depth: s.resolveNodeLimit,
+			Depth:              s.resolveNodeLimit,
+			DatastoreCallCount: 0,
 		},
 	})
+
+	grpc_ctxtags.Extract(ctx).Set("datastore_call_count", int64(resp.ResolutionMetadata.DatastoreCallCount))
```

Comment:
I'm not sure how to get that test passing. A unit test doesn't work because it doesn't setup the `grpc_ctxtags` interceptors. An integration test doesn't seem to work either because I lose access to the resulting `ctx` 🤔  i probably need to setup a test like [this](https://github.com/openfga/openfga/blob/0d97ad1996fc3dd3a00fb48b78cf0aa254f606e8/pkg/middleware/requestid/requestid_test.go)